### DISCUSSION TITLE
Is there a way to make an image fit window width?

### DISCUSSION THREAD

**Discussion by TwiggyWan** [zim-desktop-wiki/zim-desktop-wiki#2685] [General]
Hi, when I paste images, i have to right click > edit properties > set a (pixel) size to get something to display half right within my window real estate.

First, it's a bit annoying
Second, it's not responsive when i resize window or toggle side panes for example

Since zim remembers the image size (the 'edit properties' box has a 'reset size' button), It seems it would be doable to specify a percentage size instead of a pixel size.

Is there a better way to proceed? A setting or perhaps an unreleased feature? Should I convert this as an issue?

Thanks

**Comment by rpnfan** [comment]
A feature to scale images automatically would be highly appreciated!

**Reply by naturalmechanics** [reply]
Or even manually. I can't find a way to do it.

**Comment by jaap-karssenberg** [comment]
There is a "edit object" menu option when the cursor is on the image. Or
right click menu on the image.

Op zo 19 jan. 2025 05:44 schreef Savi ***@***.***>:
